ApplySyntax.sublime-settings

```diff
@@ -203,6 +203,14 @@
                 {"file_path": ".*\\.m$"},
                 {"contains": "(^|\n)\\s*%"}
             ]
+        },
+        {
+            "syntax": "TypoScript/TypoScript",
+            "rules": [
+                {"file_path": ".*(\\\\|/)ext_conf_template\\.txt$"},
+                {"file_path": ".*(\\\\|/)ext_typoscript_(setup|constants)\\.txt$"},
+                {"file_path": ".*(\\\\|/)(setup|constants)\\.txt$"}
```

Comment:
I am a little hesitant on the last rule.  `setup.txt` and `constants.txt` can very easily have conflicts.  The others seem far less likely to have conflicts.